E2E/DURABLE-FUNC-ORDER-PROCESSING/README.md

```diff
@@ -0,0 +1,125 @@
+
+![Screenshot of starting an orchestration instance through http](./img/start-orchestration-instance-http.png)
+
+3) To check the status of the orchestration instance started, go to the `statusQueryGetUri`. Your orchestration instance should show status "Running". After a few seconds, refresh to see that the orchestration instance is "Completed" and what the output is.
+
+![Screenshot of orchestration instance status](./img/orchestration-instance-status.png)
+
+### Using Visual Studio
+
+1) Open project using Visual Studio 2022 or later.
+2) Press Run/F5 to run in the debugger
+3) Use same approach above to start an orchestration instance and check its status. 
+
+### Using Visual Studio Code
+
+1) Open this folder in a new terminal
+2) Open VS Code by entering `code .` in the terminal
+3) Add a **.vscode** folder by running *"Azure Functions: Initialize project for use with VS Code"* in the Command Pallete
+4) Press Run/Debug (F5) to run in the debugger
+5) Use same approach above to start an orchestration instance and check its status. 
+
+
+## Provision the solution on Azure
+
+To set up this sample, follow these steps:
+
+1. Clone this repository to your local machine.
+2. in the root folder use the [Azure Developer CLI (azd)](https://learn.microsoft.com/azure/developer/azure-developer-cli/install-azd?tabs=winget-windows%2Cbrew-mac%2Cscript-linux&pivots=os-windows) to provision a new resource group with the environment name you provide and all the resources for the sample.
+
+```bash
+azd up
+``` 
+
+3. Set always ready instance to reduce cold start latency. The Flex Consumption plan includes this feature to let you specify a number of instances that are always running to handle requests with minimal latency. By default, this is set to 0 but can be configured in [Azure Portal](https://learn.microsoft.com/azure/azure-functions/flex-consumption-how-to?tabs=azure-portal%2Cvs-code-publish&pivots=programming-language-csharp#set-always-ready-instance-counts) or with Azure CLI. We recommend that you set always ready instance to at least 1 for Durable Functions.
+
+```bash
+az functionapp scale config always-ready set --resource-group <RESOURCE_GROUP> --name <APP_NAME> --settings durable=1
+```
+
+## Inspect the solution (optional)
+
+Once the deployment is done, inspect the new resource group. The Flex Consumption function app and plan, storage, App Insights, and networking related resources have been created and configured:
+![Screenshot of resources created by the bicept template](./img/resources-created.png)
+
+Because Durable Functions requires access to Azure Storage Blob, Table, and Queue, we created the associated networking resources such as private endpoints, link, etc. for each of those. 
+
+## Test the solution
+
+1. Go to the Azure Function app resource, find the url of the app in the top **Essentials** section of the **Overview** tab. It should look something like this: https://func-processor-abcdefgxzy.azurewebsites.net. 
```

Comment:
Just in case, optionally, Glenn has a nice way in the quickstarts that we created for finding the function URLs using the AZ CLI too:
`cli
func azure functionapp list-functions <APP_NAME> --show-keys 
`